scripts/tests/test_validate_docstrings.py

```diff
@@ -164,6 +165,8 @@ def contains(self, pat, case=True, na=np.nan):
 
         Examples
         --------
+        >>> import pandas as pd
+        >>> import numpy as np
```

Comment:
As numpy is a dependency of pandas, it makes more sense to me to import numpy first.